lib/api/filter.rb

```diff
@@ -6,7 +6,7 @@ class Filter
       ">="  => {:default => ">="},
       "<"   => {:default => "<", :datetime => "BEFORE"},
       ">"   => {:default => ">", :datetime => "AFTER"},
-      "="   => {:default => "=", :datetime => "IS", :regex => "REGULAR EXPRESSION MATCHES", :string_set => "includes all", :null => "IS NULL"},
+      "="   => {:default => "=", :datetime => "IS", :regex => "REGULAR EXPRESSION MATCHES", :string_set => "includes all", :array => "=", :null => "IS NULL"},
```

Comment:
The `:array => value` is basically a boolean here. I think all the cases are just going to use the original operator.

Does it make sense to use the values `AND` or `OR`. Those are the only real options I can see changing across `<` and others.

I'd imagine that all of these would use the same operator, but maybe `>` would require and `AND`

Otherwise, it is just acting as a boolean.

Reply:
I don't think I follow completely with your explanation here.  Can you elaborate?

* * *

That said, I was trying to keep this as small as possible to start, and iterate on it if we felt it was necessary.  Unless you think what you are suggesting might change how the end user interacts with it, I might hold of on supporting other operators with this for now.

Reply:
lol I wrote up something long here saying to ignore it.
my idea makes sense to me but not enough to elaborate.
Maybe some day over tea. Though we'd have better stuff to discuss like the weather.